repose-aggregator/services/service-client/api/src/main/java/org/openrepose/core/services/serviceclient/akka/AkkaServiceClient.java

```diff
@@ -29,6 +29,8 @@
  */
 public interface AkkaServiceClient {
 
+    void setConnectionPoolId(String connectionPoolId);
```

Comment:
yeah, we'd probably send them an email since we know it would be breaking for them. our version numbers are a little weird because all our internal changes causing major version numbers were freaking some customers out, mostly because they had heavy weight processes in place for those kind of version changes.